FishjamClient/src/main/java/com/fishjam/client/FishjamClientInternal.kt

```diff
@@ -127,28 +127,28 @@ internal class JellyfishClientInternal(
         val mediaEvent =
             PeerMessage
                 .newBuilder()
-                .setMediaEvent(MediaEvent.newBuilder().setData(event))
+                .setMediaEvent(PeerMessage.MediaEvent.newBuilder().setData(event))
                 .build()
         sendEvent(mediaEvent)
     }
 
     override fun onTrackAdded(ctx: TrackContext) {
-        val trackContext = TrackContext(ctx)
+        var trackContext = TrackContext(ctx)
```

Comment:
Why this change? `trackContext` is never updated

Reply:
ktlint ( not sure why ).